google-style.yml

```diff
@@ -148,6 +166,10 @@ Lint/NonAtomicFileOperation:
 Lint/NumberedParameterAssignment:
   Enabled: true
 
+# Can easily appear in the generated code
```

Comment:
We do run a pass of `rubocop -a` in the generator. Is this insufficient to remove cases of this and some of the others below?

Reply:
I don't see the benefit of doing this as I barely agree that having `+= 0` in the codebase is problematic.